app/helpers/trends_helper.rb

```diff
@@ -27,7 +27,7 @@ def trend_title(trend)
     end
   end
 
-  def chart_data(trend, from = Setting.max_trend, to = Time.zone.now)
+  def chart_data(trend, from = Setting.max_trend, to = Time.now.utc)
```

Comment:
It's not actually used in the trend counter retrieval, but if it was, then UTC would be correct.